lib/index.js

```diff
@@ -1,44 +1,28 @@
-module.exports = {
-  extends: [
-    'eslint:recommended',
-  ],
-  parserOptions: {
+const globals = require("globals")
+const js = require("@eslint/js")
+const pluginNode = require("eslint-plugin-node")
```

Comment:
Use `eslint-plugin-n` instead, its the currently supported fork and maintained by us in the official `eslint-community`: https://github.com/eslint-community/eslint-plugin-n